libs/types-common/src/Util/Options.hs

```diff
@@ -79,7 +79,12 @@ urlPort u = do
 makeLenses ''AWSEndpoint
 
 newtype FilePathSecrets = FilePathSecrets FilePath
-  deriving (Eq, Show, FromJSON)
+  deriving (Eq, Show, FromJSON, IsString)
+
+initCredentials :: (FromJSON a) => FilePathSecrets -> IO a
```

Comment:
I'm wondering if we couldn't make potential callers' lifes easier by using `MonadIO` here :thinking: